src/downloads/qml/single_download.cpp

```diff
@@ -367,6 +367,7 @@ void
 SingleDownload::onStarted(bool wasStarted)
 {
     m_downloading = true;
+    m_downloadInProgress = true;
```

Comment:
Thinking about it, we should also set = false once download is done

Reply:
I'm not sure but I think that's already implemented here:
https://github.com/ubports/ubuntu-download-manager/blob/6f2529a58be84d90f62ed99a735f41361596b8c5/src/downloads/qml/single_download.cpp#L335